api/queries_comments.go

```diff
@@ -99,6 +103,27 @@ func CommentUpdate(client *Client, repoHost string, params CommentUpdateInput) (
 	return mutation.UpdateIssueComment.IssueComment.URL, nil
 }
 
+func CommentDelete(client *Client, repoHost string, params CommentDeleteInput) (error) {
+	var mutation struct {
+		DeleteIssueComment struct {
```

Comment:
1 clarification. The mutation mention `issue` in the name. However, it works both for issues and PRs. Is it OK to use for both? Or for PRs should I use something else?